checker/tests/determinism/Issue110.java

```diff
@@ -0,0 +1,42 @@
+import org.checkerframework.checker.determinism.qual.*;
+
+public class Issue110 {
```

Comment:
Add tests for where:
1. The argument is `null`.  
2. The type of the argument overrides toString and its return type is `@NotDet` and `@PolyDet`. 
3. A supertype of the argument overrides toString and its return type is `@NotDet`, `@PolyDet`, and `@Det`.